pypesto/objective/history.py

```diff
@@ -1205,6 +1207,23 @@ def get_time_trace(
         """See `HistoryBase` docstring."""
         return self._get_hdf5_entries(TIME, ix)
 
+    def _check_file_id(self, file: str):
+        """
+        Check, whether the id is already existent in the file.
+
+        Parameters
+        ----------
+        file:
+            HDF5 file name.
+        """
+        with h5py.File(file, 'a') as f:
```

Comment:
Why `'a'`? To check that it's actually writable? If so, potential exceptions should be handled.